src/transformerList.ts

```diff
@@ -124,6 +124,9 @@ function docLinkFromHeadingID(headingID: string): string {
   return `https://docs.google.com/document/d/1NZA9gxtu6jD3M-5SQyx0tvV2N5qYKMgRm1XUwMnLgJU/edit#heading=${headingID}`;
 }
 
+// All possible types, used in user-configurable contracts.
+const allTypes = ["Any", "String", "Number", "Boolean", "Boundary"];
```

Comment:
There's a way of extracting this list from the `CodapLanguageType`, right? @stardust66 @P-bibs 

If we can, we could have that be the source of truth for what types exist and not duplicate it here.

Reply:
I pushed a possible fix. You have to do it the other way around, declaring the array and the deducing the type from that. Further, the array has to be constant, which means I had to sprinkle some `readonly`s around. If we decide we don't like the fix I can revert it.

Reply:
Looks good to me! I don't have a super strong preference between the two.